### PR TITLE
Image

### PR DESCRIPTION
**Pull request by vanshitalavange**
Created Image component of the following types :
- responsive image
- round image

https://user-images.githubusercontent.com/86543154/152227439-abb66824-122a-442e-9648-1ac85ea95ea5.mp4

### CODE REVIEW

**Review by vijaytembugade — APPROVED** [review]
Looks amazing. Just host on netlify, it will help to get preview.

**Review by coldpigli — COMMENTED** [review]
Not much to review here. Good to merge.

File: stylesheets/components/image.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.img-container {
+border:2px solid var(--charcoal-grey);
+padding:2rem;
+}
+.responsive-img {
+  width: 100%;
```
Comment (rahulcs754) [inline stylesheets/components/image.css:6]: If you want an image to scale down if it has to, but never scale up to be larger than its original size, use 
max-width: 100%

you also read about object-fit: cover; that will help you for responsive image

File: stylesheets/components/image.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.img-container {
+border:2px solid var(--charcoal-grey);
```
Comment (rahulcs754) [inline stylesheets/components/image.css:2]: I think you don't need to border for responsive images. if you want to border to the particular image then it is okay.

**Review by rahulcs754 — COMMENTED** [review]
Please check your code again. Read about object-fit also

File: stylesheets/components/image.css (hunk 2 above)
Comment (vanshitalavange) [reply stylesheets/components/image.css:2]: I am not giving a border to the images here. As the name suggests, the border is being given to the container that the image is residing in just to show that the image is scaling when viewed across different viewport sizes

File: stylesheets/components/image.css (hunk 1 above)
Comment (vanshitalavange) [reply stylesheets/components/image.css:6]: didn't know about `object-fit: cover`, thanks will look into this property

**Pull request merged**